src/components/displaySettings/displaySettings.template.html

```diff
@@ -143,20 +143,28 @@ <h2 class="sectionTitle">
         <select is="emby-select" class="selectSoundEffects" label="${LabelSoundEffects}"></select>
     </div>
 
-    <div class="inputContainer inputContainer-withDescription fldFadein">
```

Comment:
Outside of the scope of this PR but it's a very minor thing to spend the time opening new PR and fixing conflicts between this one. Those classes are not used in JS, so they are useless.